core/lib/dal/migrations/20231016052911_add_basic_witness_input_producer_jobs_table.up.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS basic_witness_input_producer_jobs
+(
+    l1_batch_number       BIGINT    NOT NULL PRIMARY KEY,
+    attempts              SMALLINT  NOT NULL DEFAULT 0,
+    status                TEXT      NOT NULL,
+    picked_by             TEXT,
+    input_blob_url        TEXT,
+    error                 TEXT,
+    created_at            TIMESTAMP NOT NULL,
+    updated_at            TIMESTAMP NOT NULL,
+    processing_started_at TIMESTAMP,
+    time_taken            TIME,
+    protocol_version      INT       REFERENCES protocol_versions (id)
```

Comment:
doesn't l1_batches table have the protocol version? Is it different from this one? If not, why denormalize?